pkg/testutils/jobutils/jobs_verification.go

```diff
@@ -28,6 +28,13 @@ import (
 	"github.com/pkg/errors"
 )
 
+// GetSystemJobsCount queries the number of entries in the jobs table.
+func GetSystemJobsCount(t testing.TB, db *sqlutils.SQLRunner) int {
+	var jobCount int
+	db.QueryRow(t, `SELECT COUNT(*) FROM crdb_internal.jobs;`).Scan(&jobCount)
```

Comment:
What if this query fails? Isn't there an error to check here?
I also don't think you need the `;`.

Reply:
Removed `;`. The `Scan` is from `testutils/sqlutils/sql_runner.go` which fatals on errors. It returns nothing.